src/main.rs

```diff
@@ -163,13 +164,31 @@ fn main() {
 	let map = if matches.is_present("TREE") {
 		Map::generate_three(rows, columns, initial_peek_fn, peek_fn)
 	} else if matches.is_present("PRIM") {
```

Comment:
But when using `if-else`-statements as ternary operators I quiet liked to have them on the same line as the bracket. An assignment should be concise.